eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/EnactSpec.hs

```diff
@@ -61,8 +60,9 @@ relevantDuringBootstrapSpec ::
   ConwayEraImp era =>
   SpecWith (ImpTestState era)
 relevantDuringBootstrapSpec = do
-  actionPrioritySpec
-  hardForkInitiationNoDRepsSpec
+  describe "ENACT (bootstrap phase)" $ do
+    actionPrioritySpec
+    hardForkInitiationNoDRepsSpec
```

Comment:
This is what I mean in the meeting: these tests are particularly "engineered" to do without DReps, so there is no point to run them if the DRep thresholds are non-zero (because they will fail)

Reply:
That's why we need to overwrite the thresholds for those tests, instead of completely disabling them